charts/portal/templates/deployment-backend-services.yaml

```diff
@@ -148,18 +148,18 @@ spec:
           value: "{{ .Values.backend.clients.portal }}"
         - name: "SERVICES__CATENAADMINROLES__0__USERROLENAMES__0"
           value: "{{ .Values.backend.services.catenaAdminRoles.role0 }}"
-        - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__0__DOCUMENTTYPEID"
-          value: "{{ .Values.backend.appmarketplace.type0 }}"
-        - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__0__MEDIATYPES__0"
-          value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.setting0 }}"
-        - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__1__DOCUMENTTYPEID"
-          value: "{{ .Values.backend.appmarketplace.type2 }}"
-        - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__1__MEDIATYPES__0"
-          value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.setting1 }}"
-        - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__1__MEDIATYPES__1"
-          value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.setting2 }}"
-        - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__1__MEDIATYPES__2"
-          value: "{{ .Values.backend.services.uploadServiceDocumentTypeIds.setting3 }}"
+        - name: "SERVICES__UPLOADAPPDOCUMENTTYPEIDS__0__DOCUMENTTYPEID"
```

Comment:
It needs to be UPLOADSERVICEDOCUMENTTYPEIDS in the services services.
The helm test workflow failed due to that:
https://github.com/eclipse-tractusx/portal-cd/actions/runs/5327839719/jobs/9651769407?pr=30#step:9:4397

Reply:
fixed